### ISSUE TITLE
Missing Label and Icon for Search Bar Component

### ISSUE THREAD

**Issue by PR4NJ41** [WikiEducationFoundation/WikiEduDashboard#5771]
### What is happening?
The search bar component on many pages like `explore` and `search Campaigns` is currently lacking a text label or accompanying icon, which is essential for providing clear guidance to users regarding its purpose and functionality. 

#### The search bar used earlier looked like this:
![ old](https://github.com/WikiEducationFoundation/WikiEduDashboard/assets/96368921/a122e87c-f15d-40b4-bd34-d8246edbaaa6)

#### Currently it looks like this:
![1](https://github.com/WikiEducationFoundation/WikiEduDashboard/assets/96368921/a529ef4a-0476-4f5f-9125-8f74859a6492)
![2](https://github.com/WikiEducationFoundation/WikiEduDashboard/assets/96368921/9e58c8d2-7806-454e-a350-756dbb4e2e78)

### To reproduce the behavior:

- Go to [https://outreachdashboard.wmflabs.org/explore](https://outreachdashboard.wmflabs.org/explore)
- Or go to [https://outreachdashboard.wmflabs.org/campaigns](https://outreachdashboard.wmflabs.org/campaigns)

**Comment by PR4NJ41** [comment]
@ragesoss Sir, In order to fix this, should I use icon (as earlier) or add 'search' label as in article finder?

**Comment by prathamVaidya** [comment]
@PR4NJ41 Thanks for finding this out. This issue is because of changes done on articles searchbar https://github.com/WikiEducationFoundation/WikiEduDashboard/pull/5741. I didn't know `search-bar` class inside `_article_finder.styl` was reused at other search bars too. We can either separate the CSS of article searchbar or use the same changes on all the search bars. 

If we choose to keep the new CSS. It'll only need removing some extra inline CSS in `search_bar.jsx` and adding a white svg search icon. And it will look like this :

```
<div className="search-bar" style={{ position: 'relative' }}>
      <input type="text" name={name} placeholder={placeholder} ref={ref} defaultValue={value} onKeyUp={onEnter}/>
      <button className="" onClick={onClickHandler}><i className="icon-search-white" /><span style={{ padding: '0 5px' }}>Search</span></button>
</div>
```

![image](https://github.com/WikiEducationFoundation/WikiEduDashboard/assets/61202986/23a8d879-248d-4afa-926b-07bd17721470)

**Comment by PR4NJ41** [comment]
@prathamVaidya Yes, I am aware of it. I was waiting for the final confirmation from @ragesoss before making a PR.

**Comment by ragesoss** [comment]
@PR4NJ41 good catch. I think the icon like before is best.

**Comment by PR4NJ41** [comment]
@ragesoss Thanks Sir. I will fix it. 
Also in article finder the current design should remain intact ?

**Comment by ragesoss** [comment]
> @ragesoss Thanks Sir. I will fix it. Also in article finder the current design should remain intact ?

Yes
